packages/flutter_test/lib/src/binding.dart

```diff
@@ -1218,13 +1218,15 @@ enum LiveTestWidgetsFlutterBindingFramePolicy {
   /// which can cause differences in behavior.
   fullyLive,
 
-  /// Ignore any request to schedule a frame.
+  /// Ignore any request to schedule a frame from the framework or
+  /// [WidgetTester].
   ///
   /// This is intended to be used by benchmarks (hence the name) that drive the
   /// pipeline directly. It tells the binding to entirely ignore requests for a
   /// frame to be scheduled, while still allowing frames that are pumped
   /// directly to run (either by using [WidgetTester.pumpBenchmark] or invoking
   /// [Window.onBeginFrame] and [Window.onDrawFrame]).
+  /// This is not benchmark for integration performance.
```

Comment:
This sentence isn't quite right - can you explain what you're trying to say here?

Reply:
@liyuqian https://github.com/flutter/flutter/pull/60410#discussion_r447276050

Reply:
My original suggestion was "This is not for integration performance tests". What doesn't sound quite right to me is the "This is not benchmark for" part. I felt that "This is not benchmarking for" may be more correct? If "integration performance tests" sounds strange, how about "Do not use this for integration tests (https://flutter.dev/docs/cookbook/testing/integration)."?

Reply:
I changed it to " This does not mean benchmarking performance on integration tests."